templates/quiz/meta-information.php

```diff
@@ -39,7 +40,7 @@
 	</li>
 
 	<li class="llms-quiz-meta-item llms-question-count">
-		<?php printf( __( 'Questions: %s', 'lifterlms' ), '<span class="llms-question-count">' . count( $quiz->get_questions( 'ids' ) ) . '</span>' ); ?>
+		<?php printf( __( 'Questions: %s', 'lifterlms' ), '<span class="llms-question-count">' . $quiz->get_questions_count() . '</span>' ); ?>
```

Comment:
let's improve the escaping while we're here:
```suggestion
		<?php printf( esc_html__( 'Questions: %s', 'lifterlms' ), '<span class="llms-question-count">' . esc_html( $quiz->get_questions_count() ) . '</span>' ); ?>
```

Reply:
done